static/app/types/group.tsx

```diff
@@ -391,7 +391,7 @@ export interface GroupActivityNote extends GroupActivityBase {
 }
 
 interface GroupActivitySetResolved extends GroupActivityBase {
-  data: Record<string, unknown>;
+  data: {};
```

Comment:
it didn't like that we might be rendering unknown to the dom, this removes the keys